Sources/AsyncMultiplexImage-Nuke/AsyncMultiplexImageNukeDownloader.swift

```diff
@@ -42,7 +45,17 @@ public struct AsyncMultiplexImageNukeDownloader: AsyncMultiplexImageDownloader {
       )
     )
 
-    return response
+    for task in taskMap.values {
+      task.priority = .low
+    }
```

Comment:
In SwiftUI, onDisappear did not work as expected so we can not get better timing for deprioritizing.
Instead, let the latest request be processed by making low-priority others.

Reply:
I think this is problematic because we first request a miniature image before requesting full size, so the miniature would be deprioritized

Reply:
that's true